docs/reference/js/api.md

```diff
@@ -5,10 +5,20 @@ description: Developer resources for working in JavaScript with IPFS, the InterP
 
 # IPFS in JavaScript
 
-Developers can get started with IPFS in JavaScript using Helia, a lean, modular, and modern implementation of IPFS for the JS and browser environments. Helia superseded [js-ipfs](https://github.com/ipfs/js-ipfs), which was the first JS implementation. Learn more at [https://github.com/ipfs/helia](https://github.com/ipfs/helia).
+## Helia
 
-## Hands-on examples
+Developers can get started with IPFS in JavaScript using Helia, a lean, modular, and modern implementation of IPFS for the JS and browser environments. Learn more at [https://helia.io](https://helia.io).
+
+### Hands-on examples
 
 New to IPFS or JavaScript? Get started with the [Helia 101 example](https://github.com/ipfs-examples/helia-examples/tree/main/examples/helia-101), which will walk you through spawning a Helia node, adding a file, and `cat`-ing the file CID both locally and through an IPFS gateway.
 
-Additional Helia examples can be found [here](https://github.com/ipfs-examples/helia-examples/tree/main).
\ No newline at end of file
+Additional Helia examples can be found [here](https://github.com/ipfs-examples/helia-examples/tree/main).
+
+## JS-IPFS (no-longer maintained)
+
+[JS-IPFS](https://github.com/ipfs/js-ipfs) was the first IPFS implementation in JavaScript. It still exists in its current form, but is no longer actively maintained. Please read [this post](https://github.com/ipfs/js-ipfs/issues/4336) to understand why this was done. For newer projects we recommend using [Helia](#helia) instead.
+
+### Examples
+
+JS-IPFS examples can be found [here](https://github.com/ipfs-examples/js-ipfs-examples).
```

Comment:
I think we explicitly removed mentions of js-ipfs and don't want to re-add them

Reply:
Removed mentions in https://github.com/ipfs/ipfs-docs/pull/1694/commits/b9150b343c6014dfe259a13f00e67e9878d868b9